com.sap.cloud.lm.sl.persistence/src/main/java/com/sap/cloud/lm/sl/persistence/services/AbstractFileService.java

```diff
@@ -403,6 +405,34 @@ public Integer execute(Connection connection) throws SQLException {
         return listFiles(null, namespace);
     }
 
+    public List<FileEntry> listByModificationTime(final Date modificationTime) throws FileStorageException {
+        SqlExecutor<List<FileEntry>> executor = new FileServiceSqlExecutor<List<FileEntry>>();
+        try {
+            return executor.execute(new StatementExecutor<List<FileEntry>>() {
+                @Override
+                public List<FileEntry> execute(Connection connection) throws SQLException {
+                    List<FileEntry> files = new ArrayList<FileEntry>();
```

Comment:
files -> result?

Reply:
Will remain in that way. Other functions have the same name convention.